GDJS/GDJS/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -328,11 +328,15 @@ gd::String EventsCodeGenerator::GenerateEventsFunctionContext(
          // the cost of creating/storing it for each events function might not
          // be worth it.
          "    if (objectsList) {\n" +
-         "      return parentEventsFunctionContext ?\n" +
+         "      const object = parentEventsFunctionContext ?\n" +
          "        "
          "parentEventsFunctionContext.createObject(objectsList.firstKey()) "
          ":\n" +
          "        runtimeScene.createObject(objectsList.firstKey());\n" +
+         // Add the new instance to object lists
+         "      objectsList.get(objectsList.firstKey()).push(object);"
```

Comment:
Might be a good idea to add a `if (object) { } ` around these two lines, there is a possibility the returned object is null if the name is invalid?